src/components/core/PivotTable.tsx

```diff
@@ -430,21 +434,25 @@ export class PivotTableInner extends BaseVisualization<IPivotTableInnerProps, IP
     private getColumnIds = (columns: Column[]): string[] =>
         columns.map((column: Column) => column.getColId());
 
-    private getColumnWidths = (columns: Column[]): IColumnWidths => {
+    private getResizedColumns = (columns: Column[]): IResizedColumns => {
+        const initialValue: IResizedColumns = {};
         return columns.reduce((acc, col) => {
             return {
                 ...acc,
-                [this.getColumnIdentifier(col.getDefinition() as IGridHeader)]: col.getActualWidth(),
+                [this.getColumnIdentifier(col.getDefinition() as IGridHeader)]: {
+                    width: col.getActualWidth(),
+                    agGridColId: col.getColId(),
```

Comment:
is using `getColId` safe?

Reply:
it should be safe if it is on public API of column, but it is not constant. ColId for one particular column changes during table lifetime

Reply:
i think colId changed after every sort change

Reply:
I think this should be fine for this purpose, as the `renderVisualization` generates the map with updated `colIds` each time.